DataFormats/FEDRawData/src/classes_def.xml

```diff
@@ -16,6 +16,9 @@
   <version ClassVersion="11" checksum="2009823250"/>
   <version ClassVersion="10" checksum="120358665"/>
  </class>
+ <class name="BSTRecord" ClassVersion="10">
+  <version ClassVersion="10" checksum="3262583400"/>
```

Comment:
@slava77 sorry, I was not aware of this updated numbering scheme (I looked at the other classes' versioning).
Changing.